test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
Do not commit files that are not related to the project. Instead, add them to the .gitignore file.
You committed your `test.md` to your repo. This is not a good practice.

Follow this article(https://stackoverflow.com/questions/2047465/how-can-i-delete-a-file-from-a-git-repository/2047477) on how to remove a file or a folder from git.